src/actions/publish.js

```diff
@@ -396,13 +409,15 @@ export function profileSanitized(
  */
 export function profilePublished(
   hash: string,
+  profileName: string | null,
```

Comment:
Is there a reason for making these profileName parameters nullable? I think we are only passing null value on tests. If this is for testing purpose would be good to mention it in the comments.

Reply:
Ah, I think that's because when I wrote that initially in July, `getProfileName` could return `null`, but Greg changed that since then. I'll double-check and revert this change.